core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/StatusApiServiceSpec.scala

```diff
@@ -94,37 +94,6 @@ class StatusApiServiceSpec extends ApiServiceSpec with Eventually {
     }
   }
 
-  // Ignored due to PROD-791, which stubs groups status checks to always return ok
-  it should "return 500 for non-ok status for critical subsystem" ignore withConstantCriticalErrorTestDataApiServices {
-    services =>
-      eventually {
-        withStatsD {
-          Get("/status") ~>
-            services.sealedInstrumentedRoutes ~>
-            check {
-              assertResult(StatusCodes.InternalServerError, responseAs[StatusCheckResponse]) {
-                status
-              }
-              assertResult(
-                StatusCheckResponse(
-                  false,
-                  AllSubsystems.map {
-                    case GoogleGroups =>
-                      GoogleGroups -> SubsystemStatus(false, Some(List("Could not find group: my-favorite-group")))
```

Comment:
Maybe switch to a different subsystem here instead of losing this test coverage entirely?